index.js

```diff
@@ -3,22 +3,33 @@ var postcss = require('postcss'),
 
 module.exports = postcss.plugin('postcss-rgb-plz', function (opts) {
 
+    var isValidHex = function( hex ){
+      // if it's not 3 or 6 digits, drop it.
+      if( hex.length-1 !== 3 && hex.length-1 !== 6 )
```

Comment:
If you wanted to simplify this check you could turn this statement into an expression and return the result.

```javascript
var isValidHex = function( hex ) {
  return hex.length-1 === 3 && hex.length-1 === 6;
};
```